lib/pages/my_reports_pages/my_reports_page.dart

```diff
@@ -579,206 +574,204 @@ class _MyReportsPageState extends State<MyReportsPage> {
-                                                      )
-                                                    : Container(),
-                                                Expanded(
-                                                  flex: 2,
-                                                  child: Style.body(
-                                                      report.responses![index]!
-                                                                  .answer !=
-                                                              'N/A'
-                                                          ? report
-                                                                  .responses![
-                                                                      index]!
-                                                                  .answer!
-                                                                  .startsWith(
-                                                                      'question')
-                                                              ? MyLocalizations.of(
-                                                                  context,
-                                                                  report
-                                                                      .responses![
-                                                                          index]!
-                                                                      .answer)
-                                                              : report
+                                          report.responses![index]!
+                                                      .question !=
+                                                  null
+                                              ? Expanded(
+                                                  flex: 3,
+                                                  child: Style.titleMedium(
+                                                      report
+                                                              .responses![
+                                                                  index]!
+                                                              .question!
+                                                              .startsWith(
+                                                                  'question')
+                                                          ? MyLocalizations.of(
+                                                              context,
+                                                              report
                                                                   .responses![
                                                                       index]!
-                                                                  .answer
+                                                                  .answer)
```

Comment:
`.question)` here, no?

Reply:
yeah I got confused by the diff, I have reverted the commit